animations/lib/src/basics/08_animated_list.dart

```diff
@@ -0,0 +1,118 @@
+// Copyright 2020 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+
+class AnimatedListDemo extends StatefulWidget {
+  static String routeName = '/basics/08_animated_list';
+
+  _AnimatedListDemoState createState() => _AnimatedListDemoState();
+}
+
+class _AnimatedListDemoState extends State<AnimatedListDemo> {
+  final GlobalKey<AnimatedListState> _listKey = GlobalKey();
+  List<UserModel> initialListData = listData;
```

Comment:
These two things should have their names swapped:

`final listData = initialListData`

And line 97 should be:

`List<UserModel> initialListData = [`

The list at the bottom of the file is the initial (starting) data, so it should have that name.